examples/custom-dev-node/src/main.rs

```diff
@@ -10,13 +10,13 @@ use alloy_primitives::{b256, hex};
 use futures_util::StreamExt;
 use reth::{
     builder::{NodeBuilder, NodeHandle},
-    providers::CanonStateSubscriptions,
-    rpc::api::eth::helpers::EthTransactions,
     tasks::TaskManager,
 };
-use reth_chainspec::ChainSpec;
+use reth_ethereum::{
+    chainspec::ChainSpec, node::EthereumNode, provider::CanonStateSubscriptions,
+    rpc::api::eth::helpers::EthTransactions,
+};
 use reth_node_core::{args::RpcServerArgs, node_config::NodeConfig, primitives::SignedTransaction};
```

Comment:
let 's also re-export this in a follow up from here for both reth-ethereum and reth-op:

https://github.com/paradigmxyz/reth/blob/19f8070565bd4bedc4b06d113b4fbc249410c875/crates/ethereum/reth/src/lib.rs#L79-L86